lib/elixir/lib/calendar/iso.ex

```diff
@@ -232,6 +234,22 @@ defmodule Calendar.ISO do
   def days_in_month(_, month) when month in [4, 6, 9, 11], do: 30
   def days_in_month(_, month) when month in 1..12, do: 31
 
+  @doc """
```

Comment:
What do you think about using the module variable in the `@doc` and `@spec`.

```elixir
@doc """
Returns how many months there are in the given year.

## Example

    iex> Calendar.ISO.months_in_year(2004)
    #{@months_in_year}

"""
@impl true
@since "1.7.0"
@spec months_in_year(year) :: unquote(@months_in_year)
def months_in_year(_year) do
  @months_in_year
end
```

or maybe just remove the module variable and use 12 directly like it's done in `days_in_month/2`.

```elixir
@doc """
Returns how many months there are in the given year.

## Example

    iex> Calendar.ISO.months_in_year(2004)
    12

"""
@impl true
@since "1.7.0"
@spec months_in_year(year) :: 12
def months_in_year(_year), do: 12
```

Reply:
Looks good to me too!